helmchart/sdp/values.yaml

```diff
@@ -144,8 +144,8 @@ sdp:
       SMS_SENDER_TYPE: DRY_RUN
       RECAPTCHA_SITE_KEY: #required
       CORS_ALLOWED_ORIGINS: "*"
-      ENABLE_RECAPTCHA: "false"
-      ENABLE_MFA: "false"
+      DISABLE_RECAPTCHA: "false"
+      DISABLE_MFA: "false"
```

Comment:
I assume we'll need a PR for kube to change these configmaps ? https://github.com/search?q=repo%3Astellar%2Fkube%20ENABLE_MFA&type=code

Reply:
Opened it already in https://github.com/stellar/kube/pull/1887